demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>I'm looking forward to <a href="https://github.com/stellargraph/stellargraph/issues/1454" target="_blank">https://github.com/stellargraph/stellargraph/issues/1454</a>!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I've opened https://github.com/stellargraph/stellargraph/pull/1619